directives.md

```diff
@@ -0,0 +1,14 @@
+### UCSF Website Directives
+
+
+ - UCSF Interactive branding colors used with ratio guidelines
+ - Title that is unique and specific for each page
+ - Images have appropriate alt tags
+ - Videos have accurate captions and user controls
+ - Audio files accompanied by transcripts
```

Comment:
@McKristian Are you sure that we need User Controls? Perhaps there is an easier way? :smile: